jme3-core/src/main/java/com/jme3/input/FlyByCamera.java

```diff
@@ -32,62 +32,88 @@
 package com.jme3.input;
 
 import com.jme3.collision.MotionAllowedListener;
-import com.jme3.input.controls.*;
+import com.jme3.input.controls.ActionListener;
+import com.jme3.input.controls.AnalogListener;
+import com.jme3.input.controls.KeyTrigger;
+import com.jme3.input.controls.MouseAxisTrigger;
+import com.jme3.input.controls.MouseButtonTrigger;

```

Comment:
Not an issue, but so you know, star imports do not consume more memory or performance. It's really juste a matter of taste. Also it depends on the IDE settings, mine will use star import as long as there are at least 3 imports from the same package... Maybe at some point we should settle on this and make a clear statement in the contribution guilde line, and reject PR that don't follow the rule. I personally don't mind, but every single PR has a change in import style :p

Reply:
I understand and would support a more detailed set of style guidelines ... along with better enforcement of the guidelines we have.